SphereEversion/ToMathlib/MeasureTheory/ParametricIntervalIntegral.lean

```diff
@@ -202,7 +202,7 @@ theorem hasFDerivAt_parametric_primitive_of_contDiff' {F : H → ℝ → E} (hF
     filter_upwards with t t_in
     rw [nnabs_coe K]
     exact F_lip t t_in
-  · exact integrableOn_const.mpr (Or.inr measure_Ioo_lt_top)
+  · exact (integrableOn_const_iff).mpr (Or.inr measure_Ioo_lt_top)
```

Comment:
This works, but you should be able to use `integrableOn_const` and drop the `mpr` and the `Or.inr`.